web2/htdocs/js/events.js

```diff
@@ -165,6 +165,87 @@
             });
           });
       }) /* }}} */
+      /* }}} */
+
+      /* "Pause Job" links (href="pause:...") */
+      .on('click', 'a[href^="pause:"], button[rel^="pause:"]', function (event) { /* {{{ */
+        event.preventDefault();
+        var uuid;
+        if ($(event.target).is('button')) {
+          uuid = $(event.target).attr('rel');
+        } else {
+          uuid  = $(event.target).closest('a[href^="pause:"]').attr('href');
+        }
+        uuid = uuid.replace(/^pause:/, '');
+
+        api({
+          type: 'POST',
+          url:  '/v2/tenants/'+AEGIS.current.uuid+'/jobs/'+uuid+'/pause',
+          success: function () {
+            banner('Pausing');
```

Comment:
Switch this to the affirmative/complete, and add some detail: `Paused Scheduled Backup Job ($schedule)` or something like that.